packages/ckeditor5-ckbox/src/ckboximageedit.ts

```diff
@@ -27,6 +28,6 @@ export default class CKBoxImageEdit extends Plugin {
 	 * @inheritDoc
 	 */
 	public static get requires() {
-		return [ CKBoxImageEditEditing, CKBoxImageEditUI ] as const;
+		return [ CKBox, CKBoxImageEditEditing, CKBoxImageEditUI ] as const;
```

Comment:
Maybe it should require `CKBoxEditing`? Wouldn't it be enough?

Reply:
Yes, i've checked and you're right.